### ISSUE TITLE
Train SyntaxNet model

### ISSUE THREAD

**Issue by Ldpe2G** [tensorflow/models#140]
I am trying to train the Syntaxnet model in a different language using the datasets available at http://universaldependencies.org/ and following the tutorial.

And got the following error:

F syntaxnet/term_frequency_map.cc:62] Check failed: ::tensorflow::Status::OK() == (tensorflow::Env::Default()->NewRandomAccessFile(filename, &file)) (OK vs. Not found: brain_pos/greedy/0/label-map)

Shouldn't the train step will create these files using the training dataset? 

Thanks.

**Comment by dsindex** [comment]
https://github.com/dsindex/syntaxnet/blob/master/README.md

i made some scripts for training universal treebank.
hope it does help~

**Comment by Ldpe2G** [comment]
Thank you very much! @dsindex

**Comment by Ldpe2G** [comment]
It works! I am start training now. Thank you again @dsindex

**Comment by shushengyang** [comment]
@Ldpe2G, how do you solve this problem ? I try all answer that I can find, but they doesn‘t work.
